### ISSUE TITLE
Load balancer rule not selected when clicking its row in autoscale group creation form

### ISSUE THREAD

**Issue by winterhazel** [apache/cloudstack#9991]
##### ISSUE TYPE
 * Bug Report

##### COMPONENT NAME
~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
Tested on 4.20.0.0, probably on 4.19 as well
~~~

##### CONFIGURATION
N/A

##### OS / ENVIRONMENT
N/A

##### SUMMARY
In the autoscale group creation form, if a user tries to select a load balancer rule by clicking its corresponding row (outside the radio button), the LB rule does not get selected. Instead, the user has to click inside the radio button. For other similar listings, clicking outside the radio button works.

##### STEPS TO REPRODUCE

~~~
1. Create a network
2. Create a load balancer rule for the autoscale group
3. Open the autoscale group creation form
4. Select the network 
3. Try to select a LB rule by clicking its row (not the radio button)
~~~

##### EXPECTED RESULTS

~~~
LB rule gets selected
~~~

##### ACTUAL RESULTS

~~~
LB rule does not get selected
~~~

**Comment by DaanHoogland** [comment]
again @bernardodemarco what release will you address this for , 4.20.1 or 21?

**Comment by bernardodemarco** [comment]
> again @bernardodemarco what release will you address this for , 4.20.1 or 21?

Since it's a bug fix, I think that I'll address it on the 4.20 branch.

**Comment by bernardodemarco** [comment]
Closing this ticket since it was fixed by #10016.
